Devscord.DiscordFramework/Services/EmbedMessageSplittingService.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Devscord.DiscordFramework.Services
+{
+    public class EmbedMessageSplittingService
+    {
+        private const int MAX_NUMBER_OF_MESSAGE_FIELDS = 20;
+        private MessagesService _messagesService;
+
+        public EmbedMessageSplittingService(MessagesService messagesService)
+        {
+            _messagesService = messagesService;
+        }
+
+        public async Task SendEmbedSplitMessage(string title, string description, IEnumerable<KeyValuePair<string, string>> values)
+        {
+            var messages = SplitMessage(values.ToList());
+            await _messagesService.SendEmbedMessage(title, description, messages[0]);
+            for (int i = 1; i < messages.Count; ++i)
```

Comment:
czemu for a nie foreach

Reply:
zauważ, że dla pierwsze elementu z kolekcji messages trzeba wyświetlić tytuł i opis, a dla pozostałych tylko same responses się wypisuje. Z foreach byłoby trzeba wtedy zrobić wewnątrz blok if-else, gdzie warunkiem ifa musiałoby być sprawdzanie czy mamy doczynienia z pierwszym elementem tej kolekcji czy z następnym. Przez to kod byłby trochę dłuższy.

Reply:
rozumiem, że zamiast `for (int i = 1; i < messages.Count; i++)` ma być `foreach(var message in messages.Skip(1))`. Dla mnie nie robi to wielkiej różnicy, ale ok.